modules/ai/__init__.py

```
@@ -19,9 +19,9 @@
             exclude_from="QQBot")
 
 
-@ai.command("<question> [--llm <llm>] {{I18N:ai.help}}",
+@ai.command("<prompt> [--llm <llm>] {{I18N:ai.help}}",
             options_desc={"--llm": "{I18N:ai.help.option.llm}"})
-async def _(msg: Bot.MessageSession, question: str):
+async def _(msg: Bot.MessageSession, prompt: str):
```

Ruff (0.14.1)

Unused function argument: `prompt`

(ARG001)